crates/shim/src/common.rs

```diff
@@ -0,0 +1 @@
+pub type ShutdownFn = Box<dyn FnOnce() + Send + Sync>;
```

Comment:
Could you pls align these changes with changes from https://github.com/containerd/rust-extensions/pull/41

Reply:
changed